package.json

```diff
@@ -33,7 +33,8 @@
     "coveragePathIgnorePatterns": [
       "/node_modules/",
       "<rootDir>/examples"
-    ]
+    ],
+    "testURL": "http://localhost/"
```

Comment:
Instead of adding this can you update `amex-jest-preset` to 5.0.0? It no longer uses `jsdom` as its test environment which is fine since we don't need it in this package and will solve the `localStorage` bug with Jest.